Keyneez/Keyneez-AOS
# [UI/#15] signup / 회원가입 뷰 구현

## 관련 이슈
- closed #15 

## 작업 내역
- signup 뷰 페이저 구현
- danal guide 레이아웃 구현
- danal complete 레이아웃 구현
- test guide 레이아웃 구현
- tendency 레이아웃 구현
- interest 레이아웃 구현
- test complete 레이아웃 구현
- signup otp 레이아웃 구현
- signup otp confirm 레이아웃 구현
- signup otp 레이아웃 구현
- signup otp confirm 레이아웃 구현
- tendency 버튼 구현
- linked hash set 구현
- otp 버튼 구현
- 버튼 연결

## PR 포인트
- SignupOtpFragment와 SignupOtpConfirmFragment에서 냅다 뷰모델을 재사용해봤는데 괜찮은 건지 모르겠습니다 ^__^ 일단 잘 돌아가네요ㅎㅎ
- TestCompleteFragment에서 젤리와 젤리 배경은 아직 이미지를 전달받지 못해 완전하지 못한 상태입니다
- 바텀 시트 다이얼로그는 나중에 한번에 구현하겠습니다!
- interestFragment에서 스크롤뷰를 추가했습니다.

## 실행 화면
https://user-images.githubusercontent.com/70993562/211178545-89a465bc-fd5b-4e28-a1fe-ce814e6d026f.mp4
